dojo/tools/nessus_was/parser.py

```diff
@@ -42,10 +42,10 @@ def get_findings(self, filename, test: Test):
             if not title:
                 continue
             # get description, mitigation, impact etc. from respective columns
-            description = row.get('Synopsis')
-            mitigation = str(row.get('Solution'))
-            impact = row.get('Description')
-            references = row.get('See Also')
+            description = row.get('Synopsis', '')
+            mitigation = str(row.get('Solution', ''))
+            impact = row.get('Description', '')
```

Comment:
these 2 attributes are not mandatory in the model. Are you sure you prefer blank string instead of just letting them as `NULL` is the DB?
I think it could be better to change the code as `row.get('Description')` which produce `None` as data